pyrogram/client/methods/messages/send_message.py

```diff
@@ -115,6 +115,9 @@ def send_message(
                             [InlineKeyboardButton("Docs", url="https://docs.pyrogram.org")]
                         ]))
         """
+        
+        if disable_web_page_preview is None:
```

Comment:
You are missing a few other places where `disable_web_page_preview` is used. Please search for all occurrences and add this.